tck/src/main/resources/tests/schemaTests.csv

```diff
@@ -100,3 +100,6 @@
 # testWithArgumentNameAndDefaultValue
 52|type Query          |   allHeroesIn(city: String = "New York, NY"): [SuperHero] |   Missing default value on argument
 53|type Query          |   allHeroesInTeam(team: String): [SuperHero]              |   Missing argument on Query
+
+# testSourceFieldInType
+53|type SuperHero      |   currentLocation(arg0: SuperHeroInput): String           |   Expecting a currentLocation field in SuperHero due to Source annotation
```

Comment:
I don't think this is right, but it might be good to get Bojan's thoughts on it to confirm.

My understanding of `@Source` is that the the query becomes a field on the sourced type.  So in this case, `SuperHero` has a field of `currentLocation`.  There shouldn't be a parameter from the client's point of view, because the "parameter" is the super hero itself.  

If we wanted to make this a separate query, then maybe it would go under `type Query` as:
`currentLocation(arg0: SuperHeroInput): String`.

But inside the `type SuperHero`, I think the syntax should be (suggested edit):

```suggestion
53|type SuperHero      |   currentLocation: String           |   Expecting a currentLocation field in SuperHero due to Source annotation
```

Reply:
Yea I was not sure either, as my implementation does not work yet either. (Actually, it does now, but it's creating the argument) . I looked at an old schema generated by SPQR and it contained the argument, so I based it on that. Does your generated schema not include that ?

Reply:
It did - I changed the inclusion strategy to match what is in SPQR now and I get the argument in the field/query now.  It still seems a little strange to me for it to be there though since the client never passes a value to that field to get the currentLocation.  

We can push it in now and then discuss on Friday's meeting if that makes sense.

The ID for the test should be 54 though, not 53.
```suggestion
54|type SuperHero      |   currentLocation(arg0: SuperHeroInput): String           |   Expecting a currentLocation field in SuperHero due to Source annotation
```